flw/src/main/java/org/motechproject/nms/flw/domain/WhitelistEntry.java

```diff
@@ -0,0 +1,72 @@
+package org.motechproject.nms.flw.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+import org.motechproject.nms.location.domain.State;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.Unique;
+import javax.validation.constraints.NotNull;
+
+@Entity(tableName = "whitelist_entries")
+@Unique(name = "UNIQUE_STATE_CONTACT_NUMBER_COMPOSITE_IDX", members = { "state", "contactNumber" })
+public class WhitelistEntry {
+    public static final int FIELD_SIZE_10 = 10;
+
+    @Field
+    @NotNull
+    private State state;
```

Comment:
Would the whitelist entry be a 'full' state or just a statecode and a contact number can then be looked up?